[ASM] - XPSUPPORT - 2951 (EXPANDR) - Improved Fetching and Searchable Fields (#35946)
demisto/content: Packs/CortexXpanse/ReleaseNotes/1_2_3.md

#### Incident Fields

##### Xpanse Remediation Guidance

Allowed the field to be searchable and usable for dashboard graphs.

##### Xpanse Provider

Allowed the field to be searchable and usable for dashboard graphs.

##### Xpanse Alert ID

Allowed the field to be searchable and usable for dashboard graphs.

##### Xpanse Service Validation

Allowed the field to be searchable and usable for dashboard graphs.

##### Xpanse Protocol

Allowed the field to be searchable and usable for dashboard graphs.

##### Xpanse Category

Allowed the field to be searchable and usable for dashboard graphs.

##### Xpanse Cloud Asset

Allowed the field to be searchable and usable for dashboard graphs.

##### Xpanse Service Classifications

Allowed the field to be searchable and usable for dashboard graphs.

> 
> ##### Xpanse External ID
> 
> Allowed the field to be searchable and usable for dashboard graphs.
> 
> ##### Xpanse Host Name
> 
> Allowed the field to be searchable and usable for dashboard graphs.
> 
> ##### Xpanse Responsive IP Asset
> 
> Allowed the field to be searchable and usable for dashboard graphs.
> 
> ##### Xpanse Country Code
> 
> Allowed the field to be searchable and usable for dashboard graphs.
> 
> ##### Xpanse Service ID
> 
> Allowed the field to be searchable and usable for dashboard graphs.
> 
> ##### Xpanse Business Units
> 
> Allowed the field to be searchable and usable for dashboard graphs.
> 
> ##### Xpanse Certificate Asset
> 
> Allowed the field to be searchable and usable for dashboard graphs.
> 
> ##### Xpanse IP
> 
> Allowed the field to be searchable and usable for dashboard graphs.
> 
> ##### Xpanse Service Details
> 
> Allowed the field to be searchable and usable for dashboard graphs.
> 
> ##### Xpanse Attack Surface Rule Name
> 
> Allowed the field to be searchable and usable for dashboard graphs.
> 
> ##### Xpanse Asset IDs
> 
> Allowed the field to be searchable and usable for dashboard graphs.
> 
> ##### Xpanse Domain Asset
> 
> Allowed the field to be searchable and usable for dashboard graphs.
> 
> ##### Xpanse Description
> 
> Allowed the field to be searchable and usable for dashboard graphs.
> 
> ##### Xpanse Progress Status
> 
> Allowed the field to be searchable and usable for dashboard graphs.
> 
> ##### Xpanse Port
> 
> Allowed the field to be searchable and usable for dashboard graphs.

Reviewer: ```suggestion
Allows the field to be searchable and usable for dashboard graphs.

##### Xpanse Provider

Allows the field to be searchable and usable for dashboard graphs.

##### Xpanse Alert ID

Allows the field to be searchable and usable for dashboard graphs.

##### Xpanse Service Validation

Allows  the field to be searchable and usable for dashboard graphs.

##### Xpanse Protocol

Allows the field to be searchable and usable for dashboard graphs.

##### Xpanse Category

Allows the field to be searchable and usable for dashboard graphs.

##### Xpanse Cloud Asset

Allows the field to be searchable and usable for dashboard graphs.

##### Xpanse Service Classifications

Allows the field to be searchable and usable for dashboard graphs.

##### Xpanse External ID

Allows the field to be searchable and usable for dashboard graphs.

##### Xpanse Host Name

Allows the field to be searchable and usable for dashboard graphs.

##### Xpanse Responsive IP Asset

Allows the field to be searchable and usable for dashboard graphs.

##### Xpanse Country Code

Allows the field to be searchable and usable for dashboard graphs.

##### Xpanse Service ID

Allows the field to be searchable and usable for dashboard graphs.

##### Xpanse Business Units

Allows  the field to be searchable and usable for dashboard graphs.

##### Xpanse Certificate Asset

Allows the field to be searchable and usable for dashboard graphs.

##### Xpanse IP

Allows the field to be searchable and usable for dashboard graphs.

##### Xpanse Service Details

Allows the field to be searchable and usable for dashboard graphs.

##### Xpanse Attack Surface Rule Name

Allows the field to be searchable and usable for dashboard graphs.

##### Xpanse Asset IDs

Allows the field to be searchable and usable for dashboard graphs.

##### Xpanse Domain Asset

Allows the field to be searchable and usable for dashboard graphs.

##### Xpanse Description

Allows the field to be searchable and usable for dashboard graphs.

##### Xpanse Progress Status

Allows the field to be searchable and usable for dashboard graphs.

##### Xpanse Port

Allows the field to be searchable and usable for dashboard graphs.
```

#### Integrations

##### Cortex Xpanse

> - Added support for looking back and paging for fetching Xpanse alerts. This should help assist customers that have a high throughput of alerts in XSOAR.

Reviewer: ```suggestion
- Added support for looking back and paging for fetching Cortex Xpanse alerts. This helps customers who have a high throughput of alerts in Cortex XSOAR.
```
